app/src/lang/translations/en-US.yaml

```diff
@@ -2291,3 +2293,145 @@ operations:
     batch: Batch
     batch_size: Batch Size
     cache: Cache
+core_unit_identifier: Core Unit Identifier
```

Comment:
Wondering if we should nest those values, e.g. all units under `units` and others under a corresponding panel key 🤔